assets/scss/app.scss

```diff
@@ -151,3 +151,6 @@ span.postcode {
   margin-right: 0.5em;
 }
 
+#address-text {
+  height: 6em;
```

Comment:
is this the right way to do this, should you use line-height and font-size instead?